foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/HelmMigrateTemplatesSteps.java

```diff
@@ -0,0 +1,136 @@
+package com.boozallen.aissemble.upgrade.migration;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.extensions.HelmRootChartMigrationTest;
+import com.boozallen.aissemble.upgrade.migration.extensions.HelmTemplatesMigrationTest;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+public class HelmMigrateTemplatesSteps extends AbstractMigrationTest {
+
+    private final String BASE_DIRECTORY = "./target/test-classes/test-files/migration-deploy/src/main/resources/";
```

Comment:
A: There's a convenience method for grabbing files under the tests folder
```suggestion
    private final String BASE_DIRECTORY = getTestFile("migration-deploy/src/main/resources/");
```

Reply:
Good to know

- [ ] Take advantage of existing functionality